drivers/bluetooth/hci/ipc.c

```diff
@@ -22,6 +22,10 @@ LOG_MODULE_REGISTER(bt_hci_driver);
 
 #define IPC_BOUND_TIMEOUT_IN_MS K_MSEC(1000)
 
+#define USE_SLEEP_BETWEEN_IPC_RETRIES COND_CODE_0(CONFIG_SYS_CLOCK_TICKS_PER_SEC, \
```

Comment:
Suggest to add a comment why this is require / implemented?

Reply:
I added a comment